---a/net/minecraft/tileentity/TileEntityEnderChest.java
+++b/net/minecraft/tileentity/TileEntityEnderChest.java
@@ -4,16 +4,16 @@
 import net.minecraft.init.Blocks;
 import net.minecraft.server.gui.IUpdatePlayerListBox;
 
-public class TileEntityEnderChest extends TileEntity implements IUpdatePlayerListBox
-{
+public class TileEntityEnderChest extends TileEntity implements IUpdatePlayerListBox {
+
     public float field_145972_a;
     public float prevLidAngle;
     public int field_145973_j;
     private int field_145974_k;
     private static final String __OBFID = "CL_00000355";
 
-    public void update()
-    {
-        if (++this.field_145974_k % 20 * 4 == 0)
-        {
+    public void update() {
+        if (++this.field_145974_k % 20 * 4 == 0) {
             this.worldObj.addBlockEvent(this.pos, Blocks.ender_chest, 1, this.field_145973_j);
         }
 
@@ -26,84 +26,67 @@
         float f = 0.1F;
         double d1;
 
-        if (this.field_145973_j > 0 && this.field_145972_a == 0.0F)
-        {
-            double d0 = (double)i + 0.5D;
-            d1 = (double)k + 0.5D;
-            this.worldObj.playSoundEffect(d0, (double)j + 0.5D, d1, "random.chestopen", 0.5F, this.worldObj.rand.nextFloat() * 0.1F + 0.9F);
+        if (this.field_145973_j > 0 && this.field_145972_a == 0.0F) {
+            double d0 = (double) i + 0.5D;
+            d1 = (double) k + 0.5D;
+            this.worldObj.playSoundEffect(d0, (double) j + 0.5D, d1, "random.chestopen", 0.5F, this.worldObj.rand.nextFloat() * 0.1F + 0.9F);
         }
 
-        if (this.field_145973_j == 0 && this.field_145972_a > 0.0F || this.field_145973_j > 0 && this.field_145972_a < 1.0F)
-        {
+        if (this.field_145973_j == 0 && this.field_145972_a > 0.0F || this.field_145973_j > 0 && this.field_145972_a < 1.0F) {
             float f2 = this.field_145972_a;
 
-            if (this.field_145973_j > 0)
-            {
+            if (this.field_145973_j > 0) {
                 this.field_145972_a += f;
-            }
-            else
-            {
+            } else {
                 this.field_145972_a -= f;
             }
 
-            if (this.field_145972_a > 1.0F)
-            {
+            if (this.field_145972_a > 1.0F) {
                 this.field_145972_a = 1.0F;
             }
 
             float f1 = 0.5F;
 
-            if (this.field_145972_a < f1 && f2 >= f1)
-            {
-                d1 = (double)i + 0.5D;
-                double d2 = (double)k + 0.5D;
-                this.worldObj.playSoundEffect(d1, (double)j + 0.5D, d2, "random.chestclosed", 0.5F, this.worldObj.rand.nextFloat() * 0.1F + 0.9F);
+            if (this.field_145972_a < f1 && f2 >= f1) {
+                d1 = (double) i + 0.5D;
+                double d2 = (double) k + 0.5D;
+                this.worldObj.playSoundEffect(d1, (double) j + 0.5D, d2, "random.chestclosed", 0.5F, this.worldObj.rand.nextFloat() * 0.1F + 0.9F);
             }
 
-            if (this.field_145972_a < 0.0F)
-            {
+            if (this.field_145972_a < 0.0F) {
                 this.field_145972_a = 0.0F;
             }
         }
     }
 
-    public boolean receiveClientEvent(int id, int type)
-    {
-        if (id == 1)
-        {
+    public boolean receiveClientEvent(int id, int type) {
+        if (id == 1) {
             this.field_145973_j = type;
             return true;
-        }
-        else
-        {
+        } else {
             return super.receiveClientEvent(id, type);
         }
     }
 
-    public void invalidate()
-    {
+    public void invalidate() {
         this.updateContainingBlockInfo();
         super.invalidate();
     }
 
-    public void func_145969_a()
-    {
+    public void func_145969_a() {
         ++this.field_145973_j;
         this.worldObj.addBlockEvent(this.pos, Blocks.ender_chest, 1, this.field_145973_j);
     }
 
-    public void func_145970_b()
-    {
+    public void func_145970_b() {
         --this.field_145973_j;
         this.worldObj.addBlockEvent(this.pos, Blocks.ender_chest, 1, this.field_145973_j);
     }
 
-    public boolean func_145971_a(EntityPlayer p_145971_1_)
-    {
-        return this.worldObj.getTileEntity(this.pos) != this ? false : p_145971_1_.getDistanceSq((double)this.pos.getX() + 0.5D, (double)this.pos.getY() + 0.5D, (double)this.pos.getZ() + 0.5D) <= 64.0D;
+    public boolean func_145971_a(EntityPlayer p_145971_1_) {
+        return this.worldObj.getTileEntity(this.pos) != this ? false : p_145971_1_.getDistanceSq((double) this.pos.getX() + 0.5D, (double) this.pos.getY() + 0.5D, (double) this.pos.getZ() + 0.5D) <= 64.0D;
     }
-}
+}
